### ISSUE TITLE
[Question] Architecture Diagram in `Motivation` section is not aligned with current design?

### ISSUE THREAD

**Issue by fujitatomoya** [Haivision/srt-prometheus-exporter#19]
This repository does not include extended SRT statistics property for SRT gstreamer element.
That is said it would be better to update or adjust the architecture diagram described in https://github.com/Haivision/srt-prometheus-exporter#motivation to be aligned with implementation design.

**Comment by fujitatomoya** [comment]
CC: @Windrow @Windrow14

**Comment by Windrow14** [comment]
Do you mean we should add examples of using SRT Prometheus Exporter on Gstreamer? Like to include a patch into this repository, which integrates our exporter to `srtsrc` and `srtsink` elements of Gstreamer?

**Comment by fujitatomoya** [comment]
No, I mean opposite way. i was thinking that we should update the diagram that uses only SRT sockets but Gstreamer.
